src/commands/revealResource.ts

```diff
@@ -45,14 +45,13 @@ function parsePartialAzureResourceId(id: string): Partial<ParsedAzureResourceId>
 }
 
 function getResourceKindFromId(parsedId: Partial<ParsedAzureResourceId>): 'subscription' | 'resourceGroup' | 'resource' {
-    if (parsedId.subscriptionId) {
```

Comment:
The logic here was incorrect; the Azure resource ID of, e.g., an app service site includes a subscription, and was mistakenly being identified as a subscription by this logic as a result. Flipping it upside down fixes it. I also added `else`s because I like the way they look a little better for this.